modelon/impact/client/entities/experiment.py

```diff
@@ -541,7 +541,7 @@ def _get_workflow(self) -> _Workflow:
         return _Workflow.CLASS_BASED if model.get("modelica") else _Workflow.FMU_BASED
 
     def get_class_name(self) -> str:
-        """Return the model class name."""
+        """Return the model name."""
```

Comment:
Given the name of the method the docstring would be unexpected (while `model class name` is ok-ish) . Shall we **add** `get_model_name` and eventually deprecate `get_class_name` (keep for awhile anyway)?